docs/build/tutorials/nodes-and-staking/node-backup-and-restore.md

```diff
@@ -109,6 +113,75 @@ curl -X POST --data '{
 
 You should see your original NodeID. Restore process is done.
 
+## Database
+
+Normally, when starting a new node, you can just bootstrap from the scratch. However, there are situations where you would like to speed up the process by reusing an existing database. 
+
+In order to do this, you can create a backup of your database first using the same method mentioned above (either from local node or remote node using scp). 
```

Comment:
Surround scp w/ backticks to format as code. This will make sure the command jumps out of the text. Ex: `scp`.